src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -136,6 +153,18 @@ export class HuiDialogEditCard extends LitElement {
     `;
   }
 
+  private _isDisabled(): boolean {
```

Comment:
This should be a getter

Reply:
Fixed in https://github.com/home-assistant/home-assistant-polymer/pull/4112/commits/1aa9c5800abb6f68b3d3f5505d41604c1dfd9fb6